src/main/java/org/jitsi/videobridge/EndpointBase.java

```diff
@@ -0,0 +1,378 @@
+/*
```

Comment:
normally i prefer this naming scheme (with `Base` for the common base object) but we seem to typically use `Abstract` as in `AbstractEndpoint`, so maybe it'd be better to do that for consistency.